### PR TITLE
SUL23-118 | @jdwjdwjdw | style Contact Card Paragraph

### PR DESCRIPTION
**Pull request by jdwjdwjdw**
# READY FOR REVIEW

# Summary
- [SUL23-118](https://stanfordits.atlassian.net/browse/SUL23-118): FE-DEV | style Contact Card Paragraph

# Review By (Date)
- When convenient

# Steps to Test
1. Checkout branch or view preview at https://deploy-preview-56--stanford-library.netlify.app/sws-test-314-contact-card-paragraph?slug=/sws-test-314-contact-card-paragraph
2. Add contact card paragraphs (None, Branch, and Center) to a basic page, or review [preview](https://deploy-preview-56--stanford-library.netlify.app/sws-test-314-contact-card-paragraph?slug=/sws-test-314-contact-card-paragraph)
3. Confirm all looks good, including responsive
4. Review code 🥪 

# Associated Issues and/or People
- [SUL23-118](https://stanfordits.atlassian.net/browse/SUL23-118): FE-DEV | style Contact Card Paragraph

[SUL23-118]: https://stanfordits.atlassian.net/browse/SUL23-118?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SUL23-118]: https://stanfordits.atlassian.net/browse/SUL23-118?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/(public)/components/paragraph/sul-contact-card.tsx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+import Image from "next/image";
+import {ClockIcon, EnvelopeIcon, MapPinIcon, PhoneIcon } from "@heroicons/react/24/outline";
+import useLibraryHours from "@/lib/hooks/useLibraryHours";
+import Conditional from "../utils/conditional";
+import {useResizeDetector} from "react-resize-detector";
+import { ContactCardParagraph } from "@/lib/drupal/drupal";
+
+interface ContactCardProps {
+  paragraph: ContactCardParagraph
+	siblingCount?: number
```
Comment (pookmish) [inline app/(public)/components/paragraph/sul-contact-card.tsx:13]: I think there seems to be some issue with the tab/space places. I'm not sure what could be the issue here. It looks off through this whole file.

**Review by pookmish — CHANGES_REQUESTED** [review]
I didn't get to review all the file, but can you start by fixing the spacing issue.

File: app/(public)/components/paragraph/sul-contact-card.tsx (hunk 1 above)
Comment (jdwjdwjdw) [reply app/(public)/components/paragraph/sul-contact-card.tsx]: Thanks, how about now? I was having weird issues with my editor switching between default spacing sizes, so I'm guessing that was related.

**Review by pookmish — COMMENTED** [review]
This PR is very difficult to read and know what's going on. I'm going to merge this to unblock SUL content team, but I'm going to open a new PR and I want you @jdwjdwjdw to refactor is in a way that I guide you on.

**Pull request merged**